tfjs-backend-cpu/src/register_all_kernels.ts

```diff
@@ -269,15 +268,15 @@ const kernelConfigs: KernelConfig[] = [
   logicalAndConfig,
   logicalNotConfig,
   logicalOrConfig,
-  lRNConfig,
-  lRNGradConfig,
+  LRNConfig,
```

Comment:
It seems that all other config names are starting with a lower case letter. For consistency, is `lRNconfig`/`lrnConfig` better?

Reply:
There are 3 backends implementing LRN now, cpu, webgl and node. The other two (webgl and node) are using LRNConfig (Lint will complain so "// tslint:disable-next-line: variable-name" is added to mute it), so I hope to keep them consistent.